.github/workflows/deploy-main-docs.yml

```diff
@@ -0,0 +1,60 @@
+name: deploy-main-docs
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "**/docs/**"
+
+jobs:
+  deploy-docs:
+    runs-on: ubuntu-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+    env:
+      CI: true
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup node
+        uses: actions/setup-node@v2-beta
+        with:
+          always-auth: true
+          node-version: 14
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v2
+        id: yarn-cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: Install dependencies if needed
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile --pure-lockfile
+
```

Comment:
Is it necessary to install Node when we're building documentation with Python? Even the `npx vercel` command is going to fetch it's own dependencies, isn't it?

```suggestion
```

Reply:
Totally, has sense